<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDialog</class>
 <widget class="QDialog" name="ExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export File</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>621</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Export Which Channels*?</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="allRadio">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>All!</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="allShownRadio">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>All Currently Shown:</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="shownChannelsSpecLbl">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0:3,5,7</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>290</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>*All numbers above refer to channel id (not electrode id!)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="customRadio">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>The following subset:</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="exportChansLE">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>551</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Export Filename:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filenameLE">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browseBut">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>17</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Export Range:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="allScansRadio">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>All Scans</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="allScansLbl">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0-1234567</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="selectionRadio">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Selection:</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="selectionLbl">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1231-585789</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="customRangeRadio">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Custom Range:</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="fromSB">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>400</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>400</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>to</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="toSB">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>400</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Export Format:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="binRadio">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>.bin (binary, int16)</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="csvRadio">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>.csv (text, comma-separated)</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="filesizeLbl">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>(Est. file size: 124.3 MB)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="csvRealRB">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>real</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">csvFormat</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="csvInt16RB">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>90</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>int16 (decimal)</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">csvFormat</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="csvUInt16RB">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>90</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>uint16 (hex)</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">csvFormat</string>
   </attribute>
  </widget>
 </widget>
 <tabstops>
  <tabstop>filenameLE</tabstop>
  <tabstop>browseBut</tabstop>
  <tabstop>binRadio</tabstop>
  <tabstop>csvRadio</tabstop>
  <tabstop>allRadio</tabstop>
  <tabstop>allShownRadio</tabstop>
  <tabstop>customRadio</tabstop>
  <tabstop>exportChansLE</tabstop>
  <tabstop>allScansRadio</tabstop>
  <tabstop>selectionRadio</tabstop>
  <tabstop>customRangeRadio</tabstop>
  <tabstop>fromSB</tabstop>
  <tabstop>toSB</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>csvRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>csvRealRB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>csvRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>csvInt16RB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>csvRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>csvUInt16RB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="csvFormat"/>
 </buttongroups>
</ui>
